includes/polling/os/eaton-matrix.inc.php

```diff
@@ -0,0 +1,5 @@
+<?php
+
+$hardware = trim(snmp_get($device, "matConName.0", "-OQv", "TELECOM-MIB"), '" ');
```

Comment:
Can you convert these 3 to a multi get please.

Reply:
Wilco.  Won't be able to get to this until Monday, but will try to do them first thing so you guys can review at your leisure.